docs/ruleset_functions/get_SHW_equipment_type.md

```diff
@@ -0,0 +1,44 @@
+## get_SHW_equipment_type
+
+## Description: This function determines whether the shw equipment type is one of: (ELECTRIC_RESISTANCE_INSTANTANEOUS, ELECTRIC_RESISTANCE_STORAGE, GAS_STORAGE, OTHER)
+
+## Inputs:
+- **RMD**
+- **service_water_heating_equipment_id**
+
+## Returns:
+- **type**: A string, ex: type = "ELECTRIC_RESISTANCE_INSTANTANEOUS"
+
+## Function Call: None  
+
+## Data Lookup: None
+
+## Logic:
+- we need to find the following information about the SWH:
+  - tank_type - (ELECTRIC_RESISTANCE_INSTANTANEOUS, ELECTRIC_RESISTANCE_STORAGE, GAS_STORAGE, OTHER)
+- get the service water heating system: `swh_eq = get_object_by_id(service_water_heating_equipment_id,RMD)`
+- get the type of service water heating system: `shw_type = swh_eq.type`
```

Comment:
I don't see "type" as a data element in the ServiceWaterHeatingEquipment data table. Maybe I will understand as I read through the rest of this function.

Reply:
yes.  Thanks.